src/mobility/components/BicycleSheet.tsx

```diff
@@ -86,35 +90,48 @@ export const BicycleSheet = ({vehicleId: id, close}: Props) => {
                     logoUrl={brandLogoUrl}
                   />
                 </GenericSectionItem>
-              </Section>
-              <VehicleStats
-                left={
-                  (vehicle.vehicleType.propulsionType === 'ELECTRIC' ||
-                    vehicle.vehicleType.propulsionType === 'ELECTRIC_ASSIST') &&
-                  vehicle.currentFuelPercent ? (
-                    <VehicleStat
-                      svg={Battery}
-                      primaryStat={vehicle.currentFuelPercent + '%'}
-                      secondaryStat={formatRange(
-                        vehicle.currentRangeMeters,
-                        language,
-                      )}
-                    />
-                  ) : (
-                    <VehicleStat
-                      svg={Bicycle}
-                      primaryStat=""
-                      secondaryStat={t(BicycleTexts.humanPoweredBike)}
+                <GenericSectionItem>
+                  <View style={style.content}>
+                    <MobilityStats
+                      top={
```

Comment:
What do you think if we made it into an array instead? should be more flexible in case we need to add more stats in the future.

Reply:
That would make it "future proof" but possibly harder to read as it is. It could make you believe that sending an array with three elements would magically work. My vote is to leave it as it is or rename the props to something more semantic. No big deal either way.

Reply:
True, I agree that it will make it harder to read, and I'll just rename the variable instead for now.

Reply:
Updated